webpack/published-js-components.js

```diff
@@ -51,8 +51,11 @@ const publishedJSComponents = {
   TextField: "components/TextField/TextField.jsx",
   Toast: "components/Toast/Toast.jsx",
   Tooltip: "components/Tooltip/Tooltip.jsx",
-  Loader: "components/Loader/Loader.jsx",
+  Icon: "components/Icon/Icon.jsx",
```

Comment:
is it by purpose? I know some of these components already converted to ts (icon for example).
let's pull from master and mover the components you already converted to published-ts-components.js

Reply:
Some mess happened, thanks for noticing! Fixed :)